.orchestra/ci/binary-archives-hook.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+set -euo pipefail
+
+S3_CONF_FILE=$(mktemp --tmpdir tmp.s3cmd-credentials-XXXXXXXXXX)
+DOCKER_CONTEXT=$(mktemp --tmpdir -d tmp.podman-context-XXXXXXXXXX)
+function cleanup() {
+    rm -f "$S3_CONF_FILE"
+    rm -rf "$DOCKER_CONTEXT"
+}
+trap cleanup EXIT
+
+
+#
+# S3 sync of binary archives
+#
+
+DEMO_PATHS=()
+for changed_path in "${BINARY_ARCHIVES_PATH_CHANGES[@]}"; do
+    if [[ "$changed_path" = 'public/linux-x86-64/revng-distributable-public-demo/default/'* ]]; then
```

Comment:
I want to have on `s3` `revng-distributable` too, even because that's what the `Dockerfile` uses.

Reply:
Done